frontend/lib/onboarding/onboarding-step-4.tsx

```diff
@@ -71,7 +71,10 @@ export default class OnboardingStep4 extends React.Component<
         />
         <CheckboxFormField {...ctx.fieldPropsFor("agreeToTerms")}>
           I agree to the{" "}
-          <ModalLink to={routes.step4TermsModal} component={PrivacyInfoModal}>
+          <ModalLink
+            to={routes.step4TermsModal}
+            component={() => <PrivacyInfoModal />}
+          >
```

Comment:
This is the one place where my refactoring of the `PrivacyInfoModal` required an update in the justfix codebase... hopefully this is trivial enough where it won't affect things down the line?

Reply:
Noice I think it is good, thanks!

Reply:
Yup looks good!  A funny thing is that I actually use the `render` prop instead of the `component` prop when I give it a function, but I guess it doesn't actually make a difference since the function is also a component... ummmm.  Actually do you mind changing `component` prop to `render` here and in the analogous place in the norent codebase just in case it has weird side effects?  One thing I'm trying to keep constant is for component references to remain constant between renders, which this violates, but using `render` doesn't... Probably not a big deal in the long run but it would be nice to stay consistent...